src/views/MainMenu.jsx

```diff
@@ -213,7 +210,7 @@ export default function MainMenu(props) {
 
 	return (
 		rx.user && (
-			<div class="bg_main">
+			<div className="bg_main">
```

Comment:
```suggestion
			<div class="bg_main">
```
This is solid-js, not react

Reply:
yeah - super annoying every time it's picked up as React when I run it through checker... have to explicitly set it to solid-js every time and I keep forgetting the parameter.